core/include/scipp/core/reduction.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Owen Arnold
```

Comment:
Suggest to either remove authors, or keep the main authors of the code in the file (such that other devs know who to ask about it).

Does this belong into `core/`? It can't really operate on any objects from this module? On the other hand, it does not depend on anything higher level? Is `common/` a better place (is including `dim.h` required?)?

Reply:
Good point. I'll move to common.